test/yulPhaser/Mutations.cpp

```diff
@@ -179,8 +179,8 @@ BOOST_AUTO_TEST_CASE(alternativeMutations_should_choose_between_mutations_with_g
 	for (size_t i = 0; i < 10; ++i)
 	{
 		Chromosome mutatedChromosome = mutation(chromosome);
-		cCount += (mutatedChromosome == Chromosome("c") ? 1 : 0);
-		fCount += (mutatedChromosome == Chromosome("f") ? 1 : 0);
+		cCount += (mutatedChromosome == Chromosome("c") ? 1u : 0u);
+		fCount += (mutatedChromosome == Chromosome("f") ? 1u : 0u);
```

Comment:
I wonder why it's needed here while it wasn't in case of `size_t height = m_evm15 ? 0 : 1` in one of the other files...

Reply:
I am not sure, but I guess that behaviour is different for operator `+=` and `=`... It produces error without suffix

Reply:
Oh, ok. Well, it's not all that important anyway. I just thought that maybe it's not actually necessary. I thought I got this fixed in the phaser PR.